devito/operator/operator.py

```diff
@@ -133,7 +134,8 @@ class Operator(Callable):
     """
 
     _default_headers = [('_POSIX_C_SOURCE', '200809L')]
-    _default_includes = ['stdlib.h', 'math.h', 'sys/time.h']
+    _default_includes = ['stdlib.h', 'math.h', 'sys/time.h',
+                         f'{os.path.expanduser("~")}/lookuptable.h']
```

Comment:
I need help understanding what this header file is doing. Also, is this a static header file that should become a standard part of Devito? Alternately, can this be generated using the JIT infra?

Reply:
To implement the sinc interpolation I used a look up table that holds the values of the coefficients. So this header contains the look up table and the methods responsible for initialize and get the values from it.

Access to these functions is defined from the definition of symbolic functions during the generation of symbolic equations.

I updated the PR adding the lookuptable.h file, this way you can take a look if you want to.

Reply:
About using JIT to generate the code, I don't know how it could be using JIT but I will take a look.

And yes, the static header file should become a standard part of Devito.